niimpy/EDA/EDA_countplot.py

```diff
@@ -107,9 +169,11 @@ def EDA_barplot_(df, fig_title, xlabel, ylabel):
                       xaxis_title = xlabel,
                       yaxis_title = ylabel)
     
-    fig.show()  
+    return fig 
 
-def EDA_countplot(df, fig_title, plot_type = 'count', points = 'outliers', aggregation = 'group', user = None, column=None):
+def EDA_countplot(df, fig_title, plot_type = 'count', points = 'outliers',\
```

Comment:
No need to add `\` at the end of line since the arguments are inside parenthesis.

Reply:
removed \